body.tex

```diff
@@ -294,11 +294,10 @@ \section*{Array proliferation and interoperability}
 % before executing them efficiently,
 
 Ideally, operating on specialized arrays using NumPy functions or semantics would
-simply work, so that end users could write code once, and would then benefit
-from switching between NumPy arrays, GPU arrays, distributed arrays, and so
-forth, as appropriate.
-To support array operations between external array objects, NumPy
-therefore added the capability to act as a central coordination
+simply work, so that users could write code once, and would then benefit
+from switching between NumPy arrays, GPU arrays, or distributed arrays, as appropriate.
```

Comment:
Changed the meaning slightly here: the "and so forth" indicates that there are other types of arrays out there as well. Happy to take suggestions on how to make the flow better without removing that notion.